### DISCUSSION TITLE
gluestack `add` failing for pre-initialized projects

### DISCUSSION THREAD

**Discussion by aaronatbissell** [gluestack/gluestack-ui#2464] [Q&A]
We have some new developers joining the team and they are running into errors adding components to our project.

```
npx gluestack-ui add alert

ENOENT: no such file or directory, scandir '~/.gluestack/cache/gluestack-ui/example/storybook-nativewind/src/core-components/hooks'
```

I'm assuming this is because we haven't run `npx gluestack-ui init` on this system, but when trying to do that, we get the following error:

```
npx gluestack-ui init

...

gluestack-ui is already initialized in the project, use 'npx gluestack-ui help' command to continue
```

Is there some other command that should be run to set up a user's "cache" without initializing all the project files?

**Comment by vaibhk20** [comment]
Hi @aaronatbissell, thank you for bringing up the issue. If gluestack-ui is already initialized in your project, then you're all set. You can use the `npx gluestack-ui add` command to add elements. We'll look into the issue and get back to you soon.

**Reply by aaronatbissell** [reply]
Any update on this?
